configs/dask_cluster/dev_kube_cluster/worker-cpu.tmpl

```diff
@@ -4,7 +4,7 @@ spec:
   containers:
   - image: ulissigroup/kubeflow:predictions
     imagePullPolicy: Always
-    args: [dask-worker, --nthreads, '1',--nprocs,'2', --no-dashboard, --memory-limit,'4 GiB',--death-timeout, '600']
```

Comment:
Scheduling is easier if you just do one proc per pod; maybe cut everything in half?